diff --git a/node_modules/@ant-design/icons/es/icons/WindowsOutlined.js b/node_modules/@ant-design/icons/es/icons/WindowsOutlined.js
index 3e7a408..d7d3d79 100644
--- a/node_modules/@ant-design/icons/es/icons/WindowsOutlined.js
+++ b/node_modules/@ant-design/icons/es/icons/WindowsOutlined.js
@@ -12,9 +12,4 @@ var WindowsOutlined = function WindowsOutlined(props, ref) {
   }));
 };
 
-/**![windows](data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iNTAiIGhlaWdodD0iNTAiIGZpbGw9IiNjYWNhY2EiIHZpZXdCb3g9IjY0IDY0IDg5NiA4OTYiIGZvY3VzYWJsZT0iZmFsc2UiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyI+PHBhdGggZD0iTTEyMC4xIDc3MC42TDQ0MyA4MjMuMlY1NDMuOEgxMjAuMXYyMjYuOHptNjMuNC0xNjMuNWgxOTYuMnYxNDEuNmwtMTk2LjItMzEuOVY2MDcuMXptMzQwLjMgMjI2LjVsMzgyIDYyLjJ2LTM1MmgtMzgydjI4OS44em02My40LTIyNi41aDI1NS4zdjIxNC40bC0yNTUuMy00MS42VjYwNy4xem0tNjMuNC00MTUuN3YyODguOGgzODJWMTI4LjFsLTM4MiA2My4zem0zMTguNyAyMjUuNUg1ODcuM1YyNDVsMjU1LjMtNDIuM3YyMTQuMnptLTcyMi40IDYzLjNINDQzVjIwMS45bC0zMjIuOSA1My41djIyNC44ek0xODMuNSAzMDlsMTk2LjItMzIuNXYxNDAuNEgxODMuNVYzMDl6IiAvPjwvc3ZnPg==) */
-var RefIcon = /*#__PURE__*/React.forwardRef(WindowsOutlined);
-if (process.env.NODE_ENV !== 'production') {
-  RefIcon.displayName = 'WindowsOutlined';
-}
-export default RefIcon;
\ No newline at end of file
+/**![windows](data:image/svg+xml;base64,
\ No newline at end of file
diff --git a/node_modules/@ant-design/icons/es/icons/index.js b/node_modules/@ant-design/icons/es/icons/index.js
index cd4b9e6..4ae2425 100644
--- a/node_modules/@ant-design/icons/es/icons/index.js
+++ b/node_modules/@ant-design/icons/es/icons/index.js
@@ -817,7 +817,7 @@ export { default as WeiboSquareOutlined } from "./WeiboSquareOutlined";
 export { default as WhatsAppOutlined } from "./WhatsAppOutlined";
 export { default as WifiOutlined } from "./WifiOutlined";
 export { default as WindowsFilled } from "./WindowsFilled";
-export { default as WindowsOutlined } from "./WindowsOutlined";
+// export { default as WindowsOutlined } from "./WindowsOutlined";
 export { default as WomanOutlined } from "./WomanOutlined";
 export { default as XFilled } from "./XFilled";
 export { default as XOutlined } from "./XOutlined";
